frontend/src/components/pages/RegistrantExperience/Waiver/index.tsx

```diff
@@ -1,13 +1,136 @@
+    <VStack spacing={6} pt="2" align="stretch" key={index}>
+      <Text>{clause.text}</Text>
+      <RadioGroup
+        onChange={() =>
+          waiverDispatch({
+            type: WaiverActions.CLICK_OPTIONAL_CLAUSE,
+            payload: clause,
+          })
+        }
+        key={index}
+        value={String(clause.agreed)}
+      >
+        <Stack spacing={5} direction="row">
+          <Radio value="true" mb="0">
+            {/* NOTE!?: I use mb='0' because there's a margin misalignment issue with Radio component */}
+            I agree
+          </Radio>
+          <Radio value="false">
+            I{" "}
+            <Text as="span" fontWeight="bold">
+              do not
+            </Text>{" "}
+            agree
+          </Radio>
+        </Stack>
+      </RadioGroup>
+    </VStack>
+  ));
+
+  const requiredClausesJSX: ReactJSXElement[] = Array.from(
+    waiverInterface.requiredClauses,
+  ).map((clause: RequiredClauseResponse, index: number) => (
+    <ListItem key={index}>{clause.text}</ListItem>
+  ));
 
-const Waiver = (): React.ReactElement => {
   return (
     <Box>
-      <Text textStyle="displayXLarge">Camp Registration</Text>
-      <Text>Waiver</Text>
+      <VStack spacing={8} pt="1" align="stretch">
+        <Text textStyle="displayXLarge">Camp Registration</Text>
+        <Text textStyle="displayLarge">{waiverInterface.campName}</Text>
+        <Text textStyle="displayLarge" color="primary.green.100">
+          Waiver
+        </Text>
+        <Text textStyle="displayMedium" fontWeight="bold">
+          In consideration of the participation of my child/children, (the
+          “child or children“), in the Focus on Nature photography camp/workshop
+          and all activities associated therewith,
+        </Text>
+        <Text fontWeight="bold">
+          I, the undersigned parent/guardian of the child/children, agree to the
+          following terms and conditions:
+        </Text>
+      </VStack>
+
+      <VStack spacing={5} pt="10" align="stretch">
+        <Text color="primary.green.100" fontWeight="bold">
+          Identity and Waiver of Liability
+        </Text>
+
+        <OrderedList pl="10">{requiredClausesJSX}</OrderedList>
+      </VStack>
+
+      <VStack spacing={5} pt="10" align="stretch">
+        <Text fontWeight="bold">
+          By clicking the checkbox below, you agree to the terms and conditions
+          above.
+        </Text>
+        <Checkbox
+          isChecked={waiverInterface.agreedRequiredClauses}
+          onChange={() =>
+            waiverDispatch({ type: WaiverActions.ClICK_REQUIRED_CLAUSE })
+          }
+        >
+          I agree to the above identity and liability waiver conditions{" "}
+          <Text as="span" color="red">
+            *
+          </Text>
+        </Checkbox>
+      </VStack>
+
+      <VStack spacing={5} pt="10" align="stretch">
+        <Text color="primary.green.100" fontWeight="bold">
+          Additional Clauses
```

Comment:
Could we get a divider above this?

<img width="1423" alt="Screen Shot 2022-11-26 at 3 49 45 AM" src="https://user-images.githubusercontent.com/40974372/204080564-ade0de06-9ffa-4442-9ace-1beb4dfed7cd.png">

Reply:
yep